### PR TITLE
Remove deprecated packages

### PR DESCRIPTION
**Pull request by Heyrows**
## Checklist before requesting a review
- [x] I hereby declare my code ready for review.
- [ ] If it is a critical feature, I have added tests.
- [x] The CI is successful.
- [ ] If there could backward compatibility issues, it has been discussed and planned.

[OPT] N.B for the reviewer:

## [OPT] Preview

serge-taeymans-LAjbkzSQXPE-unsplash

### CODE REVIEW

**Review by danigo99 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "cypress-setup": "./cypress-setup.sh",
     "clean-e2e-projects": "git checkout e2e-projects/ && git clean -f e2e-projects/",
     "prepare": "husky install",
-    "postinstall": "lerna bootstrap --no-private && npm install --prefix e2e-projects/next",
```
Comment (MarcMcIntosh) [inline package.json:11]: don't remove the `--no-private` flag, or we'll end up with alpha versions of deprecated packages

File: package.json [hunk 2]
```diff
@@ -17,15 +17,15 @@
     "lint-staged": "lint-staged",
     "pretty": "prettier --write .",
     "prettier:check": "prettier --check .",
-    "test": "lerna run test --no-private",
+    "test": "lerna run test",
     "test:e2e": "npm run cypress-setup && start-server-and-test 'npm run dev --prefix e2e-projects/cypress-next-app' http://localhost:3000 'npm run slicemachine --prefix e2e-projects/cypress-next-app' http://localhost:9999 'cypress run'",
     "test:e2e:dev": "npm run cypress-setup && start-server-and-test 'npm run dev --prefix e2e-projects/cypress-next-app' http://localhost:3000 'npm run slicemachine --prefix e2e-projects/cypress-next-app' http://localhost:9999 'cypress open'",
     "::changeset": "npm run changeset add && git add .changeset && git commit -m '[release:changelog] update changeset'; echo 'We commited your changeset. You should defo push this master ✌️'",
     "::release": "manypkg check && npm run build && lerna publish && npm run run changeset",
-    "bump:interactive": "lerna version prerelease --preid alpha --no-push --exact --no-private",
```
Comment (MarcMcIntosh) [inline package.json:25]: same

File: package.json [hunk 3]
```diff
@@ -17,15 +17,15 @@
...
-    "bump:alpha": "lerna version prerelease --preid $npm_config_preid --no-changelog --exact --yes --no-private",
```
Comment (MarcMcIntosh) [inline package.json:26]: same

File: package.json [hunk 4]
```diff
@@ -17,15 +17,15 @@
...
+    "bump:interactive": "lerna version prerelease --preid alpha --no-push --exact",
+    "bump:alpha": "lerna version prerelease --preid $npm_config_preid --no-changelog --exact --yes",
     "publish:alpha": "lerna publish from-package --dist-tag alpha --no-verify-access --yes",
-    "bump:production": "lerna version --conventional-graduate --exact --no-private",
```
Comment (MarcMcIntosh) [inline package.json:28]: same

**Review by MarcMcIntosh — APPROVED** [review]
I guess no-one is using these any more?

**Pull request merged**